simdev/module/git/clone_stage.py

```diff
@@ -0,0 +1,137 @@
+import hashlib
+import json
+import logging
+import os
+import textwrap
+from collections import defaultdict, namedtuple
```

Comment:
import order

Reply:
What's wrong with them?
![image](https://github.com/HSE-JetBrains-department/2023_similar_dev_search_akhundov/assets/38561919/33c71409-3d4b-4b8d-a492-079703d24a9d)
![image](https://github.com/HSE-JetBrains-department/2023_similar_dev_search_akhundov/assets/38561919/eae4b15c-c04f-49b4-816a-ee0e3e369fcc)
It's the same when i attempt to rearrange them automatically